addons/payment_test/views/payment_templates.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<template id="test_payment_checkout" inherit_id="payment.checkout" priority="1">
```

Comment:
"test_payment_checkout" is a weird name, it sounds like you're testing the checkout. Don't forget that the complete XML ID would then be `payment_test.test_payment_checkout`! You could just go for "checkout" -> `payment_test.checkout`.

Reply:
Why do you need to specify a priority?